src/core/qgsmaplayer.cpp

```diff
@@ -901,6 +901,7 @@ QString QgsMapLayer::loadNamedProperty( const QString &uri, QgsMapLayer::Propert
 {
   QgsDebugMsgLevel( QString( "uri = %1 myURI = %2" ).arg( uri, publicSource() ), 4 );
 
+  QgsDebugMsgLevel( "loadNamedProperty", 2 );
```

Comment:
Is it so important?

Reply:
umm, not at all, it must be a leftover. I'll remove it.